odoo/addons/base/models/ir_actions_report.py

```diff
@@ -696,6 +696,7 @@ def _render_qweb_pdf(self, res_ids=None, data=None):
         if not data:
             data = {}
         data.setdefault('report_type', 'pdf')
+        data.setdefault('__keep_empty_lines', True)
```

Comment:
while make sense for sweb_text, here we really don't create 'pdf' really with qweb, we generate html wiith qweb, and convert it to pdf with wkhtml, so I will not enable it by default. 

We risk just to have wrong behaviour of wkhtml, hard to reproduce because html version is not the same.